x-pack/plugins/session_view/public/components/SessionView/styles.ts

```diff
@@ -6,18 +6,26 @@
  */
 
 import { useMemo } from 'react';
+import { useEuiTheme } from '@elastic/eui';
 import { CSSObject } from '@emotion/react';
 
+
 interface StylesDeps {
   height: number | undefined;
 }
 
 export const useStyles = ({ height = 500 }: StylesDeps) => {
+
+  const { euiTheme } = useEuiTheme();
+
   const cached = useMemo(() => {
     // const { colors, border, font, size } = euiTheme;
 
+    const padding = euiTheme.size.s;
+  
     const processTree: CSSObject = {
       height: `${height}px`,
+      paddingTop: padding,
```

Comment:
since you are not reusing or changing padding, you case can use it directly from the euiTheme, don't need to put in a variable:

```js
 const processTree: CSSObject = {
      height: `${height}px`,
      paddingTop: euiTheme.size.s,
    }
```